rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -276,12 +277,36 @@ object Runtime {
     )
   )
 
+  def createWithEmptyCost[F[_]: ContextShift: Concurrent: Log: Metrics, M[_]](
+      dataDir: Path,
+      mapSize: Long,
+      storeType: StoreType,
+      extraSystemProcesses: Seq[SystemProcess.Definition[F]] = Seq.empty
+  )(
+      implicit
+      P: Parallel[F, M],
+      executionContext: ExecutionContext
+  ): F[Runtime[F]] =
+    (for {
+      costAccounting <- CostAccounting.empty[F]
+      runtime <- {
+        implicit val ca: CostAccounting[F] = costAccounting
+        implicit val cost: _cost[F]        = loggingCost(ca, noOpCostLog)
```

Comment:
I believe we should be passing the `cost` as parameter, and decide whether it's logging or not at the end of the world.

Reply:
Sounds good. Let's do that in a separate PR when `CostAccounting` is gone

Reply:
ah, you're saying this thing is a functionally-dependent on costAccounting? I'm fine with that then.